composer.json

```diff
@@ -1,134 +1,55 @@
-        "ezsystems/ezplatform-ee-installer": "^3.3@dev",
-        "ezsystems/ezplatform-elastic-search-engine": "^1.2@dev",
-        "ezsystems/ezplatform-form-builder": "^2.3@dev",
-        "ezsystems/ezplatform-graphql": "^2.3@dev",
-        "ezsystems/ezplatform-http-cache": "^2.3@dev",
-        "ezsystems/ezplatform-http-cache-fastly": "^2.1@dev",
-        "ezsystems/ezplatform-icons": "~1.0.0@dev",
-        "ezsystems/ezplatform-kernel": "^1.3@dev",
-        "ezsystems/ezplatform-matrix-fieldtype": "^2.2@dev",
-        "ezsystems/ezplatform-page-builder": "^2.3@dev",
-        "ezsystems/ezplatform-page-fieldtype": "^2.3@dev",
-        "ezsystems/ezplatform-permissions": "^1.1@dev",
-        "ezsystems/ezplatform-personalization": "^1.0@dev",
-        "ezsystems/ezplatform-query-fieldtype": "^2.3@dev",
-        "ezsystems/ezplatform-rest": "^1.3@dev",
-        "ezsystems/ezplatform-richtext": "^2.3@dev",
-        "ezsystems/ezplatform-search": "^1.2@dev",
-        "ezsystems/ezplatform-segmentation": "^1.1@dev",
-        "ezsystems/ezplatform-site-factory": "^1.3@dev",
-        "ezsystems/ezplatform-solr-search-engine": "^3.3@dev",
-        "ezsystems/ezplatform-standard-design": "^0.4@dev",
-        "ezsystems/ezplatform-user": "^2.3@dev",
-        "ezsystems/ezplatform-version-comparison": "^1.3@dev",
-        "ezsystems/ezplatform-workflow": "^2.3@dev",
-        "ezsystems/ezrecommendation-client": "^2.1@dev",
-        "friendsofsymfony/jsrouting-bundle": "^2.7",
-        "gregwar/captcha-bundle": "^2.0",
-        "hautelook/templated-uri-bundle": "^3.3",
-        "ibexa/image-editor": "^1.0@dev",
-        "ibexa/migrations": "^1.0@dev",
-        "ibexa/oauth2-client": "^1.0@dev",
-        "knplabs/knp-menu-bundle": "^3.1",
-        "lexik/jwt-authentication-bundle": "^2.10",
-        "monolog/monolog": "^2.1",
-        "php-http/guzzle6-adapter": "^2.0",
-        "platformsh/symfonyflex-bridge": "^2.2",
-        "sensio/framework-extra-bundle": "^5.6",
-        "symfony/asset": "^5.2.1",
-        "symfony/cache": "^5.2.1",
-        "symfony/console": "^5.2.1",
-        "symfony/dotenv": "^5.2.1",
-        "symfony/expression-language": "^5.2.1",
-        "symfony/flex": "^1.10",
-        "symfony/form": "^5.2.1",
-        "symfony/framework-bundle": "^5.2.1",
-        "symfony/monolog-bundle": "^3.6",
-        "symfony/orm-pack": "^1.0",
-        "symfony/process": "^5.2.1",
-        "symfony/proxy-manager-bridge": "^5.2.1",
-        "symfony/security-bundle": "^5.2.1",
-        "symfony/serializer-pack": "^1.0",
-        "symfony/swiftmailer-bundle": "^3.5",
-        "symfony/thanks": "^1.2",
-        "symfony/translation": "^5.2.1",
-        "symfony/twig-bundle": "^5.2.1",
-        "symfony/validator": "^5.2.1",
-        "symfony/web-link": "^5.2.1",
-        "symfony/webpack-encore-bundle": "^1.8",
-        "symfony/yaml": "^5.2.1",
-        "twig/extra-bundle": "^3.1"
+        "symfony/flex": "^1.11.0"
+    },
+    "flex-require": {
```

Comment:
I've been thinking... What exactly `flex-require` gives us here? Execution of recipes? I'm asking because if we were to deliver it like that, it would be difficult to perform merges on custom projects. It also would raise a question - if I have my own 3rd party packages, should I move them to `flex-require` as well?

In other words: AFAIR `composer update` transforms this list into `require`, so is there anything blocking us from delivering it in such "ready" form?

POV ping @webhdx

Reply:
As discussed, I will test this solution and if it works for us then I will change it as you suggest